Sources/Tokenizers/PostProcessor.swift

```diff
@@ -139,3 +143,51 @@ class RobertaProcessing: PostProcessor {
         return text.reversed().prefix(while: { $0.isWhitespace }).count - 1
     }
 }
+
+class BertProcessing: PostProcessor {
+    private let sep: (UInt, String)
+    private let cls: (UInt, String)
+
+    required public init(config: Config) {
+        guard let sep = config.sep?.tokenValue else { fatalError("Missing `sep` processor configuration") }
+        guard let cls = config.cls?.tokenValue else { fatalError("Missing `cls` processor configuration") }
+        self.sep = sep
+        self.cls = cls
+    }
+
+    func postProcess(tokens: [String], tokensPair: [String]?, addSpecialTokens: Bool = true) -> [String] {
+        guard addSpecialTokens else { return tokens + (tokensPair ?? []) }
+
+        var outTokens = [self.cls.1] + tokens + [self.sep.1]
+        if let tokensPair = tokensPair, !tokensPair.isEmpty {
+            outTokens += tokensPair + [self.sep.1]
+        }
+
+        return outTokens
+    }
+}
+
+class SequenceProcessing: PostProcessor {
+    private let processors: [PostProcessor]
+
+    required public init(config: Config) {
+        guard let processorConfigs = config.processors?.arrayValue else {
+            fatalError("Missing `processors` configuration")
+        }
+
+        self.processors = processorConfigs.compactMap { PostProcessorFactory.fromConfig(config: $0) }
+    }
+
+    func postProcess(tokens: [String], tokensPair: [String]?, addSpecialTokens: Bool = true) -> [String] {
+        var currentTokens = tokens
+        var currentTokensPair = tokensPair
+
+        for processor in processors {
+            let processed = processor.postProcess(tokens: currentTokens, tokensPair: currentTokensPair, addSpecialTokens: addSpecialTokens)
+            currentTokens = processed
+            currentTokensPair = nil  // After the first processor, we no longer have a separate pair
+        }
```

Comment:
The implementation in [transformers.js is slightly different](https://github.com/xenova/transformers.js/commit/db9250b3e814dc43b1cb35d9c2cd0feff30e3932#diff-323fc64a0a2174a18ce315f2bc15d7ef8ac9c34e68c394f9eba64516e61aeb41R1773) cc @xenova